src/api/Ethereum.ts

```diff
@@ -132,10 +133,9 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
       let { data } = await network({
         method: "GET",
         url: URL.format({
-          pathname:
-            getEnv("NFT") && currency.ticker === "ETH"
-              ? `https://explorers.api-01.live.ledger-stg.com/blockchain/v3/eth/addresses/${address}/transactions`
-              : `${baseURL}/addresses/${address}/transactions`,
+          pathname: isNFTActive(currency)
+            ? `https://explorers.api-01.live.ledger-stg.com/blockchain/v3/eth/addresses/${address}/transactions`
```

Comment:
Why not an env for this URL?

Reply:
big temporary hack indeed. the env is "EXPLORER" in practice, the ledger-stg.com api had bugs in some other endpoints which forced us to branch only that endpoint

Reply:
we have a tech debt task logged